### PR TITLE
New feature adds ability to reorder service start and stop 

### PR DESCRIPTION
**Pull request by louismunro**
Not ready.

### CODE REVIEW

File: lib/pf/cmd/pf/service.pm [hunk 1]
```diff
@@ -88,15 +88,24 @@ our %ACTION_MAP = (
 
 our $ignore_checkup = $FALSE;
 
+sub _byIndexOrder { 
+    $a->new->orderIndex <=> $b->new->orderIndex;
+}
+
 sub parseArgs {
```
Comment (jrouzierinverse) [inline lib/pf/cmd/pf/service.pm:95]: To ensure that the sorting is "stable" add the following pragma.
use sort 'stable';

File: lib/pf/services/manager/pfsetvlan.pm [hunk 2]
```diff
@@ -14,6 +14,7 @@ pf::services::manager::pfsetvlan
 use strict;
 use warnings;
 use Moo;
+use pf::config;
```
Comment (julsemaan) [inline lib/pf/services/manager/pfsetvlan.pm]: why ?

You don't seem to be doing anything with it

Comment (louismunro) [inline lib/pf/services/manager/pfsetvlan.pm]: I thought it looked better this way.

(I forgot to remove it when I refactored this code no to need it anymore...)

**Comment by louismunro** [comment]
Review meeeeeeee.....

**Comment by fdurand** [comment]
The code looks good to me. Tested and works. Ready to merge

**Pull request merged**